.github/SUPPORT.md

```diff
@@ -17,6 +17,6 @@ For more advanced needs (e.g. CKEditor integration in your products, or developm
 
 For more reading visit our supporting resources:
 
-* An extensive [CKEditor 4 documentation](http://docs.ckeditor.com/) with information about CKEditor features and configuration settings.
+* An extensive [CKEditor 4 documentation](https://docs.ckeditor.com/ckeditor4/latest/) with information about CKEditor features and configuration settings.
```

Comment:
The link for documentation is: https://docs.ckeditor.com/ckeditor4/docs/, https://docs.ckeditor.com/ckeditor4/latest/ is the entry page that includes SDK etc. Since we have samples separately below, I'd use the direct documentation link.